apiserver/apiserver.go

```diff
@@ -1129,33 +1137,27 @@ func (srv *Server) serveConn(
 	recorderFactory := observer.NewRecorderFactory(apiObserver, nil, observer.NoCaptureArgs)
 	conn := rpc.NewConn(codec, recorderFactory)
 
-	modelService := srv.shared.serviceFactoryGetter.FactoryForModel(database.ControllerNS).Model()
-
 	// Note that we don't overwrite modelUUID here because
 	// newAPIHandler treats an empty modelUUID as signifying
 	// the API version used.
-	resolvedModelUUID := modelUUID
-	if modelUUID == "" {
-		info, err := modelService.ControllerModel(ctx)
-		if err != nil {
-			return errors.Trace(err)
-		}
-		resolvedModelUUID = info.UUID.String()
+	resolvedModelID := modelID
+	if modelID == "" {
```

Comment:
Small nit. Can we change this check to:

`if resolvedModelID == ""` That way we are keep the same the language throughout the expression.